apps/web/src/components/currency-input/currency-input.tsx

```diff
@@ -56,7 +57,10 @@ export default function CurrencyInput({
     if (onBlur) onBlur(event_)
   }
 
-  const localizedValue = formatCurrency(value, locale, currency, conversionRate)
+  const localizedValue =
```

Comment:
This makes it so it only shows the helper localized value if it's not the default currency (USD)